test/test_fmcomms5_prod.py

```diff
@@ -0,0 +1,285 @@
+                sample_rate=30720000,
+                rx_rf_bandwidth_chip_b=18000000,
+                tx_rf_bandwidth_chip_b=18000000,
+            ),
+        ),
+    ],
+)
+def test_rssi_chip_b(
+    test_gain_check,
+    iio_uri,
+    classname,
+    channel,
+    param_set,
+    dds_scale,
+    min_rssi,
+    max_rssi,
+):
+    test_gain_check(
+        iio_uri, classname, channel, param_set, dds_scale, min_rssi, max_rssi
+    )
+
+
+@pytest.mark.iio_hardware(hardware)
+@pytest.mark.parametrize("classname", [(classname)])
+@pytest.mark.parametrize("channel", [0, 1, 2, 3])
+@pytest.mark.parametrize(
+    "dds_scale, frequency, hardwaregain_low, hardwaregain_high",
+    [(0.0, 999859, 50, 80), (0.89, 999859, 0.0, 28)],
+)
+def test_hardware_gain(
+    test_hardwaregain,
+    iio_uri,
+    classname,
+    channel,
+    dds_scale,
+    frequency,
+    hardwaregain_low,
+    hardwaregain_high,
+):
+    test_hardwaregain(
+        iio_uri,
+        classname,
+        channel,
+        dds_scale,
+        frequency,
+        hardwaregain_low,
+        hardwaregain_high,
+    )
+
+
+@pytest.mark.iio_hardware(hardware)
+@pytest.mark.parametrize("classname", [(classname)])
+@pytest.mark.parametrize("channel", [0, 1, 2, 3])
+def test_fmcomms5_loopback(test_dma_loopback, iio_uri, classname, channel):
+    test_dma_loopback(iio_uri, classname, channel)
+
+
+@pytest.mark.iio_hardware(hardware)
+@pytest.mark.parametrize("classname", [(classname)])
+def test_dcxo(test_dcxo_calibration, classname, iio_uri):
+    test_dcxo_calibration(classname, iio_uri)
+
+
+@pytest.mark.iio_hardware(hardware)
+@pytest.mark.parametrize("classname", [(classname)])
+@pytest.mark.parametrize("channel", [0, 1])
+@pytest.mark.parametrize(
+    "param_set",
+    [
+        dict(
+            tx_lo=2400000000,
+            rx_lo=2400000000,
+            tx_hardwaregain_chan0=-10,
+            tx_hardwaregain_chan1=-10,
+            sample_rate=30720000,
+        )
+    ],
+)
+@pytest.mark.parametrize(
+    "low, high",
+    [([-20.0, -100.0, -110.0, -110.0, -110.0], [-10.0, -60.0, -75.0, -75.0, -80.0])],
+)
+def test_harmonic_values(
+    test_harmonics, classname, iio_uri, channel, param_set, low, high, plot=True
+):
+    test_harmonics(classname, iio_uri, channel, param_set, low, high, plot)
+
+
+@pytest.mark.iio_hardware(hardware)
+@pytest.mark.parametrize("classname", [(classname)])
+@pytest.mark.parametrize("channel", [2, 3])
+@pytest.mark.parametrize(
+    "param_set",
+    [
+        dict(
+            tx_lo_chip_b=2400000000,
+            rx_lo_chip_b=2400000000,
+            tx_hardwaregain_chip_b_chan0=-10,
+            tx_hardwaregain_chip_b_chan1=-10,
+            sample_rate=30720000,
+        )
+    ],
+)
+@pytest.mark.parametrize(
+    "low, high",
+    [([-20.0, -100.0, -110.0, -110.0, -110.0], [-10.0, -60.0, -75.0, -75.0, -80.0])],
+)
+def test_harmonic_values_chip_b(
+    test_harmonics, classname, iio_uri, channel, param_set, low, high, plot=True
```

Comment:
Do you want the plots to show up here?

Reply:
In the case of production tests I think it is relevant to be able to see the plots by default, without having to edit the code

Reply:
I will make an update soon and remove them